client/src/components/Admin/Jobs/jobComponent.vue

```diff
@@ -0,0 +1,143 @@
+                <b-col> <label>Company Name</label>
+                    <b-form-input required v-model="jobDetails.companyName" placeholder="Enter Company Name"
+                        trim></b-form-input></b-col></b-row>
+            <b-row>
+
+                <b-col> <label>Company Description</label>
+                    <b-form-textarea required id="textarea" v-model="jobDetails.companyDescription"
+                        placeholder="Enter Company Description"></b-form-textarea></b-col>
+                <b-col>
+                    <label>Address</label>
+                    <b-form-textarea required id="textarea" v-model="jobDetails.address"
+                        placeholder="Enter Address"></b-form-textarea></b-col>
+                <b-col> <label>Industry</label>
+                    <b-form-input required v-model="jobDetails.industryType"
+                        placeholder="Enter Industry"></b-form-input></b-col>
+            </b-row>
+
+            <b-row>
+                <b-col>
+                    <label>Vacancy</label>
+                    <b-form-input required v-model="jobDetails.vacancy" placeholder="Enter Vacancy"></b-form-input></b-col>
+                <b-col> <label>Employment Type</label>
+                    <b-form-input required v-model="jobDetails.employmentType"
+                        placeholder="Enter Employment Type"></b-form-input></b-col>
+                <b-col>
+                    <label>Experience Required</label>
+                    <b-form-input required v-model="jobDetails.experienceRequired"
+                        placeholder="Enter Vacancy"></b-form-input></b-col>
+
+            </b-row> <b-row>
+
+                <b-col> <label>Valid Till Date</label>
+                    <datepicker required v-model="jobDetails.validTillDate" name="uniquename"></datepicker>
+                </b-col>
+
+                <b-col>
+                    <label>Country</label>
+                    <select required v-model="countrySelected" class="custom-select custom-select-sm"
+                        aria-label="Default select example">
+                        <option value="" disabled>Select</option>
+                        <option v-for="(country, index) in countrys" :value="country.id" :key="index">{{
+                            country.name }} ({{
+        country.countryCode }})</option>
+                    </select></b-col>
+                <b-col>
+                    <label>Skills Required</label>
+                    <select v-model="skillsRequired" class="custom-select custom-select-sm"
+                        aria-label="Default select example">
+                        <option value="" disabled>Select</option>
+                        <option v-for="(skill, index) in skillsDetails" :key="index">{{ skill }}
+                        </option>
+                    </select></b-col>
+            </b-row>
+            <b-row> <b-col cols="4">
+                    <label>Status</label>
```

Comment:
make label meaningful 
Job status would be nice
also I feel status is placed at last, would be nice if you could place it somewhere more visible to user..